src/platform-implementation-js/lib/inject-script-EMBEDDED.ts

```diff
@@ -1,19 +1,17 @@
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const fs = require('fs');
+// const fs = require('fs');
 
 export function injectScriptEmbedded() {
   const url = 'https://www.inboxsdk.com/build/pageWorld.js';
 
   const script = document.createElement('script');
   script.type = 'text/javascript';
 
-  const originalCode = fs.readFileSync(
-    __dirname + '/../../../dist/pageWorld.js',
-    'utf8'
-  );
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const originalCode = require('raw-loader!../../../dist/pageWorld.js');
```

Comment:
Unfortunately, this version interestingly seems to get into a rebuild cycle sometimes. I have a feeling this affected the browserify version as well.